scanpipe/management/commands/graph.py

```diff
@@ -85,10 +83,9 @@ def add_arguments(self, parser):
 
     def handle(self, *pipelines, **options):
         if options["list"]:
-            for location, _ in scanpipe_app_config.pipelines:
-                self.stdout.write("- " + self.style.SUCCESS(location))
-                pipeline_doc = get_pipeline_doc(location)
-                self.stdout.write(indent(pipeline_doc, "  "), ending="\n\n")
+            for name, pipeline_class in scanpipe_app_config.pipelines.items():
```

Comment:
same nit as above wrt. using `pipelines`... what is a pipeline? is this a Pipeline subclass? if that's the case, then I would think that `pipelines` would be a sequence of Pipelines, `pipeline_names` or `pipeline_keys` a sequence of Pipeline keys and `pipelines_by_key` or `pipelines_by_name` mappings? 
for discussion